### PR TITLE
[FIX] migrate categories of hr.holiday.status

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: addons/hr_holidays/migrations/7.0.1.5/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@openupgrade.migrate()
+def migrate(cr, version):
+    # write categories from crm.case.categ
+    cr.execute('alter table crm_meeting_type add column %s integer' %
+               openupgrade.get_legacy_name('crm_meeting_type_id'))
+    cr.execute(
+        'insert into crm_meeting_type (name, %s) '
+        'select name, id from crm_case_categ where object_id='
+        "(select id from ir_model where model='crm.meeting')" %
+        openupgrade.get_legacy_name('crm_meeting_type_id'))
```
Comment (StefanRijnhart) [inline addons/hr_holidays/migrations/7.0.1.5/post-migration.py]: Should this not be called openupgrade.get_legacy_name('crm_case_categ_id')?

**Comment by pedrobaeza** [comment]
Please remember also to add a PR to modify migration status on docs of the 8.0 branch.

**Comment by pedrobaeza** [comment]
OK, I see it's already marked as done, so forget it.

**Comment by hbrunn** [comment]
@StefanRijnhart done

**Comment by pedrobaeza** [comment]
I have checked that crm_case_categ are not inserted twice, so everything seems OK

:+1:

**Comment by StefanRijnhart** [comment]
:+1:

**Pull request merged**